internal/services/consumer.go

```diff
@@ -71,8 +72,15 @@ func (cgh *consumerGroupHandler) Cleanup(sarama.ConsumerGroupSession) error {
 
 func (cgh *consumerGroupHandler) ConsumeClaim(session sarama.ConsumerGroupSession, claim sarama.ConsumerGroupClaim) error {
 	for message := range claim.Messages() {
-		log.Printf("Message received: value=%s, partition=%d, offset=%d", string(message.Value), message.Partition, message.Offset)
+		cm := jsonToCanaryMessage(message.Value)
+		log.Printf("Message received: value=%+v, partition=%d, offset=%d", cm, message.Partition, message.Offset)
 		session.MarkMessage(message, "")
 	}
 	return nil
 }
+
+func jsonToCanaryMessage(value []byte) CanaryMessage {
+	var cm CanaryMessage
+	json.Unmarshal(value, &cm)
+	return cm
+}
```

Comment:
How would you feel about moving this conversion out of the consumer and into canary_message.go? Combined with the comment below, it would gather all our conversions into one place.
```
func NewCanaryMessage([]byte) CanaryMessage
```